tplfunc.go

```diff
@@ -84,6 +91,26 @@ func init() {
 		}
 		return string(out)
 	}
+
+	zhttp.FuncMap["totpbarcode"] = func(email, s string) template.URL {
+		totpURI := fmt.Sprintf("otpauth://totp/GoatCounter:%s?secret=%s&issuer=GoatCounter", email, s)
+		buf := bytes.NewBufferString("data:image/png;base64,")
+		qrCode, err := qr.Encode(totpURI, qr.M, qr.Auto)
+		if err != nil {
+			panic(fmt.Sprintf("Error encoding QR Code: %q", err))
+		}
+		qrCode, err = barcode.Scale(qrCode, 200, 200)
+		if err != nil {
+			panic(fmt.Sprintf("Error scaling QR Code: %q", err))
+		}
+		e := base64.NewEncoder(base64.StdEncoding, buf)
+		err = png.Encode(e, qrCode)
+		if err != nil {
+			panic(fmt.Sprintf("Error encoding QR Code PNG: %q", err))
+		}
+		/* #nosec */
```

Comment:
What does this comment do?

Reply:
oops, sorry, left those in out of habit. It disables a tool called gosec for this line, but unless you're using it it's kind of pointless. I'll remove these.

Reply:
Nevermind, looks like one of your commits removed it so it's good!